Update js-yaml version

hnan1118 opened the pull request:
With js-yaml@3.12.0, benchmarks for Fabric-v1.4 fails while
instantiating chaincode.

Signed-off-by: Jinwoo Hwang <hnan1118@ajou.ac.kr>

panyu4 commented:
@hnan1118 ， Did you run `npm install -g node-gyp node-pre-gyp` before testing ? I don't know if the version of js-yaml caused the failure.

aklenik commented:
@hnan1118 Please share more info about the error you encountered, and why you think js-yaml was the problem.

hnan1118 commented:
Hmm, weird. I tried to generate the same errors by downgrading js-yaml but it's working fine now.

I was trying to setup caliper on my remote ubuntu machine and kept on having problem with instatiating chain code.
```
warn: [e2eUtils.js]: instantiate proposal was bad: 
Error: error starting container: error starting container:
Failed to generate platform-specific docker build:
Error uploading input to container:
API error (404): Could not find the file /chaincode/input in container 224be72dd688d3ed38855a2843f3e3c76bdab40b7afe0384ce4e1c1d6d3e1162
```
I'm not sure if this was the specific error I got, but I think this was the one.

Using Fabric-v1.1 was the only working benchmark.

Then, I tried the same on my local ubuntu laptop, and Fabric-v1.4 worked fine.
I compared the npm list with my remote machine and the only difference was js-yaml version (3.13.0 / 3.12.0)
So I updated my js-yaml version on my remote machine and it worked fine so I opened this pull request.
I'm pretty much sure the only package I changed was the js-yaml though.

aklenik commented:
@hnan1118 Looks like a transient error that failed to copy/prepare/find the installed chaincode. I don't think it's a client-side (Caliper) error

hnan1118 commented:
Okay. Thank you guys for the kind reply.
Now that I know I've mistaken, I'm closing the pull request.
